code/__HELPERS/roundend.dm

```diff
@@ -359,7 +359,10 @@
 		if(aiPlayer.mind)
 			parts += "<b>[aiPlayer.name]</b> (Played by: <b>[aiPlayer.mind.key]</b>)'s laws [aiPlayer.stat != DEAD ? "at the end of the round" : "when it was <span class='redtext'>deactivated</span>"] were:"
 			parts += aiPlayer.laws.get_law_list(include_zeroth=TRUE)
-
+		else if(aiPlayer.deployed_shell)
```

Comment:
```suggestion
		else if(aiPlayer?.deployed_shell?.mind)
```
Just a tiny improvement, makes it take less code, else if leading into an if is yucky

Reply:
aiPlayer will always be reference to object unless the global AI list got fucked up